lib/snapshot/BUILD.gn

```diff
@@ -290,3 +291,16 @@ compile_platform("strong_platform") {
 group("kernel_platform_files") {
   public_deps = [ ":strong_platform" ]
 }
+
+if (is_win) {
+  zip_bundle("archive_win_gen_snapshot") {
```

Comment:
Who is the caller of this target definition? Is this patch complete?

Reply:
This was for experimenting with building the windows archives but we are currently blocked on arm support.